src/panels/lovelace/elements/types.ts

```diff
@@ -7,6 +7,8 @@ import type { HuiImage } from "../components/hui-image";
 interface LovelaceElementConfigBase {
   type: string;
   style?: Record<string, string>;
+  left?: number;
+  top?: number;
```

Comment:
We should handle this entirely in the editor instead of adding redundant config options. The editor should translate between the `style` option and the top/left inputs